cli/args/flags.rs

```diff
@@ -2997,11 +2982,7 @@ fn permission_args_parse(flags: &mut Flags, matches: &clap::ArgMatches) {
     flags.allow_ffi = Some(vec![]);
     flags.allow_hrtime = true;
   }
-  #[cfg(not(test))]
-  let has_no_prompt_env = env::var("DENO_NO_PROMPT") == Ok("1".to_string());
```

Comment:
Env var is now resolved in `CliOptions` (multiple source resolution) to keep this file strictly about argument parsing.